File: frontend/src/entity/ReplyDTO.ts
export interface ReplyDTO {
    replyId:number;
    content:string;
    postId:number;
    displayName:string;
    username:string;
    avatar?:string|null;
    replyTime:Date;
    parentReplyId?:number|null;
    replyToDisplayName?:string|null;
}

File: backend/src/main/java/com/ucaacp/backend/entity/DTO/ReplyDTO.java
package com.ucaacp.backend.entity.DTO;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

import java.time.LocalDateTime;

@Data
@NoArgsConstructor
@AllArgsConstructor
public class ReplyDTO {

    private Integer replyId;
    private String content;
    private Integer postId;
    private String displayName;
    private String username;
    private String avatar;
    private LocalDateTime replyTime;
    private Integer parentReplyId;
    private String replyToDisplayName;


}
